packages/flutter/lib/src/widgets/image.dart

```diff
@@ -152,12 +154,15 @@ class Image extends StatefulWidget {
     this.alignment,
     this.repeat: ImageRepeat.noRepeat,
     this.centerSlice,
-    this.gaplessPlayback: false
+    this.gaplessPlayback: false,
+    this.package
   }) : image = new FileImage(file, scale: scale),
        super(key: key);
 
   /// Creates a widget that displays an [ImageStream] obtained from an asset
-  /// bundle. The key for the image is given by the `name` argument.
+  /// bundle. The key for the image is given by the `name` argument. If used in
+  /// a package implementation, the `package` argument must be non-null
+  /// otherwise it should be omitted.
```

Comment:
include the same information here as earlier regarding what package does, how to use it, etc.